docs/rules/periphery/periphery-split-csv.py

```diff
@@ -73,6 +74,72 @@ class RuleTable:
     def csv_fname(self):
         return 'p{:03d}-{}.csv'.format(self.order, self.fname)
 
+def pick_units (rdesc, rvalue):
+    """Picks rule unit based on rule description and value
+       Units are detected by presence of any alias or by keyword guess.
+       Optionally, keyword within first 2 words can override unit alias found (strong guess).
+       Description is filtered: unit aliases are unified, trailing '[unit]' removed.
+
+    Args:
+        rdesc: str 
+            rule description
+        rvlue: str 
+            rule value
+
+
+    Returns:
+        unit : str
+            rule unit 
+        desc: str 
+            filtered rule description
+    """
+    mu, sq = '\u00B5', '\u00B2'
+    no_unit_values = {'N/A':'N/A','':''}    # values which imply no unit
+    desc_picks = {  # recognized units and their aliases
```

Comment:
Alignment here is a bit weird?

Reply:
We will add the Value column to the relevant CSV's as it was originally planned